genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -697,10 +697,10 @@ func TestAccResourceTfExportUserPromptExportAudioFile(t *testing.T) {
 			// Update to two resources with separate audio files
 			{
 				Config: gcloud.GenerateUserPromptResource(&gcloud.UserPromptStruct{
-					userPromptResourceId,
-					userPromptName,
-					strconv.Quote(userPromptDescription),
-					userPromptResources2,
+					ResourceID:  userPromptResourceId,
```

Comment:
Refactored this because Go best practice is to explicitly have key names in your struct definition.  Caught this via go vet.